packages/pharos-site/src/components/SearchBox.tsx

```diff
@@ -14,7 +14,7 @@ const SearchBox: FC = () => {
       <div className={searchBox__container}>
         <input className={searchBox__input} placeholder="Search" />
         <div className={searchBox__icon}>
-          <PharosIcon name="search" />
+          <PharosIcon name="search" a11yHidden="true" />
```

Comment:
@daneah Do you know if adding additional attributes to pharos site components will cause issues until the unreleased changes are released and Pharos site is updated to use it? Should we not release the new pharos site until it has the next release of Pharos? (🤔 if that is the case does this become a major release for pharos site, since it is "breaking"?)

Seems a bit of a chicken/egg situation here so just trying to figure the best way to manage that, or at least how it was handled in the past?

Reply:
This is a good question—normally I'd say components should be accepting of unrecognized attributes, but I'm not actually confident here whether e.g. TypeScript will fail to compile if it can detect you're passing an unrecognized thing. If the site preview builds on this PR (checking now), I think it might all work out okay.